.github/workflows/elm.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         with:
           activate-environment: elm-xonsh-test
           environment-file: ci/environment-elm.yml
-          update-conda: true
+          auto-update-conda: true
```

Comment:
@scopatz is this ok/needed? I don't know too much about conda

Other than that I think this PR is ready to merge 🔥

Reply:
without it would show warning. That miniconda action has renamed this variable long ago